Update README.

MancunianSam opened the pull request:
The README had instructions for building the lambda zip manually which is now done by Jenkins. I've replaced it with a description of each Jenkins job and what it does.

MancunianSam committed:
Update README.

The README had instructions for building the lambda zip manually which
is now done by Jenkins. I've replaced it with a description of each
Jenkins job and what it does.

suzannehamilton commented on README.md:
```diff
@@ -2,81 +2,28 @@ This is the code and configuration to carry out the antivirus checks on a single
+| Dockerfile-dependencies | yara-dependencies | Installs necessary software on an amazon linux image and zips it up to be used by the lambda                              |
 
-`docker cp rules:/rules/output ./lambda`
+The build job rebuilds all of these images. This isn't necessary most of the time because the dependency versions and yara version don't change that often and so this job is only ever run manually when we need to update the dependencies.
```
Thanks for adding these explanations! Could you add some deployment instructions as well?

e.g. "To deploy everything, run https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Antivirus%20Build/. If you just need to deploy changes to the lambda, run https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Antivirus%20Deploy/. (Assuming that's true? And you don't need to run any of the intermediate steps to do the tagging?)

MancunianSam replied:
Good point, I've added a section on deploying the images. I've also added in the `--no-cache` and `--pull` arguments to the build commands so we get the latest versions every time.

MancunianSam committed:
Add a section about deploying changes

MancunianSam committed:
Add no-cache and pull

This means that whenever we run the build job, it will get the latest
base image. This should allow us to fix most vulnerabilities in the
image just by running this job.

suzannehamilton commented on README.md:
````diff
@@ -2,81 +2,42 @@ This is the code and configuration to carry out the antivirus checks on a single
-docker stop rules dependencies
-```
+### Python code updates
+This includes our own custom code and any dependencies in requirements.txt This shouldn't need any manual intervention as it should trigger the [test](https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Antivirus%20Test/) job which in turn runs the bundle and deploy jobs. You can run the test job manually if you need to for any reason. 
````
Does this only apply to integration? How do you deploy changes to  staging?

MancunianSam replied:
It's just a case of running the deploy job, I've added in a line to say this.

MancunianSam committed:
Add section for deploying to other environments

MancunianSam committed:
Merge branch 'master' into update-readme
